.github/CODEOWNERS

```diff
@@ -21,6 +57,15 @@ v2/spanner-migrations-sdk/ @GoogleCloudPlatform/spanner-migrations-team
 v2/spanner-custom-shard/ @GoogleCloudPlatform/spanner-migrations-team
 v2/spanner-to-sourcedb/ @GoogleCloudPlatform/spanner-migrations-team
 
+# Spanner testing infra
+it/google-cloud-platform/src/main/java/org/apache/beam/it/gcp/spanner @GoogleCloudPlatform/dataflow-templates-wg
```

Comment:
Should this point to spanner-migrations-team?

Reply:
Oops, yes - thanks